ReactWindows/ReactNative/Views/Modal/ReactModalShadowNode.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using ReactNative.UIManager;
+
+namespace ReactNative.Views.Modal
+{
+    /// <summary>
+    /// The shadow node implementation for modal views.
+    /// </summary>
+    public class ReactModalShadowNode : LayoutShadowNode
+    {
+        /// <summary>
+        /// Instantiates the <see cref="ReactModalShadowNode"/>.
+        /// </summary>
+        public ReactModalShadowNode()
+            : base(false, true)
```

Comment:
> true [](start = 25, length = 5)

Did you use "delegated layout" on purpose? It would make me happy, that would mean there's more than one usage scenario for that.

Reply:
I had originally used it on purpose, but I don't recall exactly why, so I did some testing and it seems it's not required anymore. Sorry for getting your hopes up hah